tests/zfs-tests/tests/functional/persist_l2arc/setup.ksh

```diff
@@ -0,0 +1,45 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
+# Use is subject to license terms.
+#
+
+#
+# Copyright (c) 2013, 2016 by Delphix. All rights reserved.
```

Comment:
This file is not "All Rights Reserved" copyrighted by "Sun Microsystems inc."
Delphix might have code here, if thats the case this is fine. Otherwise get rid of it.

Required header ends at `CDDL HEADER END`, the copyright notice below that is not a legal requirement anymore since quite a few years. However: Even if you use it, the copyright notice should coincide with the people actually having committed code, which in this case is not sun.

Same goes for (some) other files as well.

Reply:
Yes, of course. I copied them from the cache test but forgot to modify the headers. Thanks for catching it!

Reply:
No problem, legal OCD was kicking in 👍